packages/jest-runtime/src/index.js

```diff
@@ -130,6 +134,8 @@ class Runtime {
     this._mockRegistry = Object.create(null);
     this._moduleMocker = this._environment.moduleMocker;
     this._moduleRegistry = Object.create(null);
+    this._nativeModuleRegistry = Object.create(null);
+    this._path = path || '<unknown test file>';
```

Comment:
Can you change this to "unknown entry point"? `jest-runtime` can be used as standalone and doesn't have to run test files.